sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/ActiveDirectoryGroupImpl.java

```diff
@@ -133,18 +168,17 @@ public ActiveDirectoryGroupImpl withEmailAlias(String mailNickname) {
             // domainName = parts[1]; // no use
             mailNickname = parts[0];
         }
-        createParameters.withMailNickname(mailNickname);
+        innerModel().withMailNickname(mailNickname);
         return this;
     }
 
     @Override
     public ActiveDirectoryGroupImpl withMember(String objectId) {
-        membersToAdd
-            .add(
-                String.format(
-                    "%s%s/directoryObjects/%s",
-                    manager().serviceClient().getEndpoint(), manager().tenantId(),
-                    objectId));
+        // https://docs.microsoft.com/en-us/graph/api/group-post-members
+        String membersKey = "@odata.id";
+        membersToAdd.add(new HashMap<>() {{
```

Comment:
Maybe `Collections.singletonMap` would be better (if it is immutable)? `{{` seems to give a subclass of `HashMap`.

Reply:
done